tests/qa_automation/execute_test_run.pm

```diff
@@ -21,8 +21,13 @@ use upload_system_log;
 use base "opensusebasetest";
 
 sub run {
-    my $timeout = get_var("MAX_JOB_TIME") || 7200;
-    my $test    = get_var("QA_TESTSUITE") . get_var("QA_VERSION");
+    my $timeout   = get_var("MAX_JOB_TIME") || 7200;
```

Comment:
We can use get_var($variable [, $default ]) here, or // to put default value. I know that || will work, but it's rather logical operator.

Reply:
Thanks, I'll take it, similar use, easy to modified.

Reply:
New verification run:
http://10.67.133.102/tests/37

Reply:
The verify failed because network in Beijing site very slow this days, zypper install process times out. But http://10.67.133.102/tests/34 was good, I only change two minor part after that as mitiao and rwx788's recommend.